src/schemas/f142/f142_rw.cxx

```diff
@@ -78,19 +92,31 @@ append_ret writer_typed_array<DT, FV>::write_impl(FBUF const *fbuf) {
 }
 
 template <typename DT, typename FV>
-writer_typed_scalar<DT, FV>::writer_typed_scalar(
-    hid_t hdf_group, std::string const &sourcename) {
+writer_typed_scalar<DT, FV>::writer_typed_scalar(hid_t hdf_group,
+                                                 std::string const &sourcename,
+                                                 Value fb_value_type_id)
+    : fb_value_type_id(fb_value_type_id) {
```

Comment:
The member variable should not be named the same as the constructor argument. You were using an underscore prefix for member variables in some places, that would solve the problem here.